providers/yaml/src/main/java/org/jboss/resteasy/plugins/providers/YamlProvider.java

```diff
@@ -84,6 +90,11 @@ public boolean isReadable(Class<?> type, Type genericType, Annotation[] annotati
    public Object readFrom(Class<Object> type, Type genericType, Annotation[] annotations, MediaType mediaType,
                            MultivaluedMap<String, String> httpHeaders, InputStream entityStream) throws IOException,
             WebApplicationException {
+      // Set "org.yaml.snakeyaml.allow-all-tags" to true to make snakeyaml parsing work in permissive mode, allowing all tags
+      // to be parsed. The tags will be checked later by a resteasy provided constructor (TypeSafeConstructor).
+      // This is only relevant for snakeyaml 1.33.SP2+, which is Red Hat fork of snakeyaml 1.33.

```

Comment:
I would remove this sentence and point to the upstream discussion on this

Reply:
Yes, that's a shame. Could we point to https://bitbucket.org/snakeyaml/snakeyaml/pull-requests/44 ?